src/scenes/PreloaderScene.js

```diff
@@ -13,10 +13,9 @@ class PreloaderScene extends Scene {
     this.load.image('load-game-btn', 'assets/images/load-game-btn.png');
     this.load.image('title-hill1', 'assets/images/title-hill1.png');
     this.load.image('dummy-platform', 'assets/images/dummy/dummy-platform.png');
-    this.load.spritesheet('mc-africa', 'assets/images/sprites/mc-africa.png', {
-     frameWidth: 244,
-     frameHeight: 287
-    });
+
+    this.load.multiatlas('mc-africa', 'assets/images/sprites/mc-africa/packed/mc-africa.json', 'assets/images/sprites/mc-africa/packed');
```

Comment:
First time I've seen this, looks cool I will read up more on it when I get a bit of time! :+1: